.github/workflows/add-to-projects.yml

```diff
@@ -108,3 +108,12 @@ jobs:
           project-url: https://github.com/orgs/camunda/projects/79
           github-token: ${{ secrets.GH_PROJECT_AUTOMATION_TOKEN }}
           labeled: component/feel-js
+      - id: add-to-devops
+        name: Add to Monorepo DevOps Team project
+        uses: actions/add-to-project@v1.0.2
+        if: ${{ steps.has-project.outputs.result == 'false' }}
+        with:
+          project-url: https://github.com/orgs/camunda/projects/115
```

Comment:
let's keep an eye on whether that token `GH_PROJECT_AUTOMATION_TOKEN` has enough permissions to add to this project.

Reply:
The token is working as tested in https://github.com/camunda/camunda/issues/31253#event-17358724889 - it belongs to `@megglos` though, so not the cleanest automation...